__tests__/app.test.js

```diff
@@ -97,6 +97,36 @@ describe("GET /api/articles",()=>{
             })
         })
     })
+    test("responds with status 200 an article object of id provided with comment_count",()=>{
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.article).toHaveProperty('comment_count')
+            expect(body.article).toEqual(expect.objectContaining({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(Number)
+            }))
+            expect(body.article).toEqual({
+                article_id: 1,
+                title: 'Living in the shadow of a great man',
+                topic: 'mitch',
+                author: 'butter_bridge',
+                body: 'I find this existence challenging',
+                created_at: '2020-07-09T20:11:00.000Z',
+                votes: 100,
+                article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+                comment_count: 11
```

Comment:
Rather than amending a previous test, we should leave it as it was, then make a new test to show that comment_count has been added now. The first expect statement here with objectContaining would be fine, and there's no requirement to use the second one with toEqual 👍